manifests/server/config.pp

```diff
@@ -117,15 +117,15 @@
   # Open read permissions to private keys to puppet group for foreman, proxy etc.
   file { "${::puppet::server::ssl_dir}/private_keys":
     ensure  => directory,
-    owner   => $::puppet::server::user,
-    group   => $::puppet::server::group,
+    owner   => $::puppet::server_private_keys_owner,
```

Comment:
changed. Though now I'm realizing I actually want to manage this directory on all nodes, not just puppetserver nodes